### ISSUE TITLE
Screen readers cannot explore forms without setting them to error states

### ISSUE THREAD

**Issue by gnomeontherun** [vmware/clarity#3557]
## Describe the bug

A screen reader cannot interact with the form (focus to read and blur on exit) without triggering errors.

## How to reproduce

https://stackblitz.com/github/clane/clarityTest2/tree/master/clarity-test-app

- Using VoiceOver with Safari on Mac OS read the field in the Helper and validation message example
- Advance the VoiceOver cursor to continue reading the page (Ctr + Option + right arrow key)
- Move the reading cursor back to the input field (Ctr + Option + left arrow key)
- Observe that you hear the error message 

## Expected behavior

Provide hidden text warning screen reader users of this behavior

## Versions

**App**

* Angular: [e.g. 6]
* Node: [e.g. 8.10.0]
* Clarity: [e.g. 0.12.5]

**Device:**

* Type: [e.g. MacBook]
* OS: [e.g. iOS]
* Browser [e.g. chrome, safari]
* Version [e.g. 22]

## Additional notes

We have to decide if we can do this automatically or expect applications to do this. There may not be any validation on the form which would make a message confusing.
